indico/modules/events/templates/management/event_person_links.html

```diff
@@ -0,0 +1,16 @@
+{% from 'forms/_form.html' import simple_form %}
+{% from 'events/management/_event_person_links.html' import render_event_person_links %}
+
+{% set title = _('speakers') if event.type == 'lecture' else _('chairpersons') %}
```

Comment:
don't do this for i18n. it won't be correct in some other languages. For example, in german the proper title to manage speakers would be would be "Redner verwalten", but the way you build the data-title you could only put in the wrong order (manage=verwalten).